File: Shared/Dto/R2D2C/ClientMeasurementDoneDto.cs
using System.Runtime.Serialization;

namespace Iit.Fibertest.Dto
{
    [DataContract]
    public class ClientMeasurementDoneDto
    {
        [DataMember]
        public string ClientIp { get; set; }

        [DataMember]
        public ReturnCode ReturnCode { get; set; }

        [DataMember]
        public byte[] SorBytes { get; set; }
    }
}

File: Client/WebClient/src/app/models/dtos/port/clientMeasurementDoneDto.ts
import { ReturnCode } from "../../enums/returnCode";
import { RtuMaker } from "../../enums/rtuMaker";

export class ClientMeasurementDoneDto {
  connectionId: string;
  clientIp: string;
  returnCode: ReturnCode;
  sorBytes: number[];
  clientMeasurementId: string;
  rtuMaker: RtuMaker;
}
